### PR TITLE
Add support for Parblo Ninos N7B

### PR DESCRIPTION
**Pull request by nefir2**

### CODE REVIEW

File: OpenTabletDriver.Configurations/OpenTabletDriver.Configurations.csproj [hunk 1]
```diff
@@ -26,6 +26,16 @@
...
+  <ItemGroup>
+    <None Remove="Configurations\Parblo\Ninos N7B.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <AdditionalFiles Include="Configurations\Parblo\Ninos N7B.json">
+      <TabletConfiguration>true</TabletConfiguration>
+    </AdditionalFiles>
+  </ItemGroup>
+
```
Comment (gonX) [inline OpenTabletDriver.Configurations/OpenTabletDriver.Configurations.csproj:38]: This change is unnecessary
```suggestion
```
The file should already be getting picked up by the itemgroup a few lines above

File: OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+  "Name": "Parblo Ninos N7",
```
Comment (gonX) [inline OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json:2]: Tablet name should match the file name.
```suggestion
  "Name": "Parblo Ninos N7B",
```

File: OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  "Specifications": {
+    "Digitizer": {
+      "Width": 177.8,
+      "Height": 114.0,
+      "MaxX": 28799.0,
+      "MaxY": 16199.0
```
Comment (gonX) [inline OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json:8]: Are you sure this is correct? This is usually calculated based off LPI (usually 100 or 200) with `MaxX`/`MaxY` as reference point. You can get `MaxX` and `MaxY` with the help of the Tablet Debugger in OpenTabletDriver.

If these MaxX and MaxY values are correct, perhaps `Width` is `143.995` and `Height` is `80.995`?

File: OpenTabletDriver.Configurations/OpenTabletDriver.Configurations.csproj [hunk 4]
```diff
@@ -26,6 +26,16 @@
     <AdditionalFiles Include=".\Configurations\**\*.json" TabletConfiguration="true" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Configurations\Parblo\Ninos N7B.json" />
```
Comment (jamesbt365) [inline OpenTabletDriver.Configurations/OpenTabletDriver.Configurations.csproj:30]: Why are you touching this?

File: OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json (hunk 2 above)
Comment (jamesbt365) [inline OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json:2]: Correct this.

File: OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      "InputReportLength": 1101,
+      "OutputReportLength": 1101,
+      "ReportParser": "OpenTabletDriver.Configurations.Parsers.XP_Pen.XP_PenReportParser",
+      "FeatureInitReport": null,
+      "OutputInitReport": [
+        "ArAE"
+      ],
+      "DeviceStrings": {},
+      "InitializationStrings": []
+    }
```
Comment (gonX) [inline OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json:46]: You should be able to skip one of these identifiers, likely the latter one if I had to guess

**Review by gonX — CHANGES_REQUESTED** [review]

**Review by jamesbt365 — CHANGES_REQUESTED** [review]
Please amend these changes and follow our [CONTRIBUTING.md](https://github.com/OpenTabletDriver/OpenTabletDriver/blob/master/CONTRIBUTING.md) thanks.

File: OpenTabletDriver.Configurations/OpenTabletDriver.Configurations.csproj (hunk 4 above)
Comment (nefir2) [reply OpenTabletDriver.Configurations/OpenTabletDriver.Configurations.csproj:30]: Visual Studio did this automatically

**Comment by gonX** [comment]
Don't forget to add an entry to [TABLETS.md](https://github.com/OpenTabletDriver/OpenTabletDriver/blob/master/TABLETS.md). You can see how to properly do that in [CONTIBUTING.md](https://github.com/OpenTabletDriver/OpenTabletDriver/blob/master/CONTRIBUTING.md#adding-a-configuration)

Please confirm the following with the Tablet Debugger:
- [ ] Pressure reaches the full range
- [ ] Max values reached in matches MaxX/MaxY
- [ ] Both tilt axises are working
- [ ] All buttons work, on both tablet and pen

**Comment by jamesbt365** [comment]
Once done it would be nice if you were to squash commits ^^

**Comment by nefir2** [comment]
> Don't forget to add an entry to [TABLETS.md](https://github.com/OpenTabletDriver/OpenTabletDriver/blob/master/TABLETS.md). You can see how to properly do that in [CONTIBUTING.md](https://github.com/OpenTabletDriver/OpenTabletDriver/blob/master/CONTRIBUTING.md#adding-a-configuration)
> 
> Please confirm the following with the Tablet Debugger:
> 
>     * [ ]  Pressure reaches the full range
> 
>     * [ ]  Max values reached in matches MaxX/MaxY
> 
>     * [ ]  Both tilt axises are working
> 
>     * [ ]  All buttons work, on both tablet and pen

 * [x]  Pressure reaches the full range

 * [x]  Max values reached in matches MaxX/MaxY

 * [ ]  Both tilt axises are working

 * [x]  All buttons work, on both tablet and pen

I don't know how to check the third point because I don't know how it works. I can see "Tilt" in "Properties" section in the Debugger, but it changes values randomly when I use the stylus.

**Comment by jamesbt365** [comment]
Tilt corresponds to which direction the pen is tilted.

**Comment by gonX** [comment]
If you hold your pen from the very end of it and let it dangle naturally by gravity (like a pendulum), the tilt reading should be close to `[0, 0]`. Make sure the pen is reasonably close to the tablet as some tablets do not report tilt unless it is fairly close to the surface.

One number should be corresponding to the horizontal axis while the other number should correspond to the vertical axis.

**Comment by gonX** [comment]
Hello, please let us know how you're progressing with this. We're very interested in adding support for this tablet.

File: OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json (hunk 3 above)
Comment (X9VoiD) [reply OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json:8]: This doesn't look right. If current values are true then the sensors aren't a square.

**Comment by jamesbt365** [comment]
Fixes #3187.

**Review by jamesbt365 — APPROVED** [review]
Unless i'm missing anything, which I might be as it has been a while since I've looked into this tablet and the N7, I'm assuming the sizing is a little different between these 2 models as your config differs from the other.

Shouldn't be anything else stopping a merge of this one?

**Pull request merged**